site/pages/servicecomplexe/index-fr.hbs

```diff
@@ -43,11 +43,11 @@
 		<h2>[Sous-titre]</h2>
 		<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ipsa hic minima magni nesciunt nam. Facilis perferendis, adipisci voluptates, enim illo perspiciatis molestias itaque iste qui distinctio asperiores quidem voluptatum sunt.</p>
 		</section>
-		<nav role="navigation" class="row pagntn-prv-nxt">
+		<nav role="navigation" class="row">
 			<h2 class="wb-inv">Navigation dans le document</h2>
 			<ul class="list-unstyled">
 				{{#next}}
-				<li class="col-sm-6 col-sm-push-6 text-right"><a rel="next" href="{{link}}"><span class="glyphicon glyphicon-chevron-right"></span><span class="pgntn-lbl">Suivant</span> {{title}}</a></li>
+				<li><div class="col-xs-12"><a href="http://wet-boew.github.io/themes-dist/GCWeb/servicecomplexe/page2-fr.html" class="btn btn-primary"><span></span>Suivant<span class="wb-inv"> proceed to the next page</span></a></div></li>
```

Comment:
No need for absolute links as the relative link value is in the `{{link}}` variable.